### DISCUSSION TITLE
Need a comment box while starring your favorite repos!

### DISCUSSION THREAD

**Discussion by Curovearth** [community/community#38326] [Other Feature Feedback, Questions, & Ideas]
**Concept of giving a ⭐ to our loved repos**
- We usually star repositories which are helpful to us or if it's an awesome open source contribution.
- The update of distributing your stars into particular list was so good.

**The problem I face**
- I star the repos and place it to my desired list. Now after coming back to the particular repository after a month makes me sometimes wonder why did I actually star it?

Here's the **Idea** 💡
- Incorporating a comment box while starring a repo such that the user can layout specific reasons in pointers or anything so that it can be referred when coming back. Thus, helping users like me, who don't have to scratch their head on repositories which were starred more than 3 months back or a year as well 😂
- Your reasons can stay there in the box until you feel to remove them or you can have the option of updating them as your work grows around the repository.

_A comment box or a git-reason box, whatever name sounds good!_

Do Vote! ⬇

**Comment by RuslansAleksejevs** [comment]
This
